### PR TITLE
[Quickfix] Force flash message width to 80% of screen on small devices

### PR DESCRIPTION
**Pull request by ghost**
Flash messages are too large on phones.

### CODE REVIEW

File: app/assets/stylesheets/_flash_messages.scss [hunk 1]
```diff
@@ -19,4 +19,11 @@
     &.alert-succes { border: 1px solid darken($state-success-bg, 10%); }
     &.alert-warning { border: 1px solid darken($state-warning-bg, 10%); }
   }
+
+  @media (max-width: $grid-float-breakpoint-max) {
+    .flash-message {
+      max-width: 80%;
+      min-width: 80%;
```
Comment (denschub) [inline app/assets/stylesheets/_flash_messages.scss]: `min-width` + `max-width` = `width`?

**Comment by AugierLe42e** [comment]
Override [of the original rules](https://github.com/AugierLe42e/diaspora/blob/fix-flash-message-width/app/assets/stylesheets/_flash_messages.scss#L15).

File: app/assets/stylesheets/_flash_messages.scss (hunk 1 above)
Comment (AugierLe42e) [inline app/assets/stylesheets/_flash_messages.scss]: Override [of the original rules](https://github.com/AugierLe42e/diaspora/blob/fix-flash-message-width/app/assets/stylesheets/_flash_messages.scss#L15).

Comment (denschub) [inline app/assets/stylesheets/_flash_messages.scss]: Mh. Meh. Looks odd, but fine. :)

**Comment by jhass** [comment]
Merged as e731bb86c570a8202045379f26d654a6e73bca7f

Thanks

**Pull request closed without merging**